modules/rosa-sts-cluster-terraform-destroy.adoc

```diff
@@ -28,40 +27,61 @@ before destroying your resources. These variables are matched to resources to de
 ----
 $ terraform destroy
 ----
-ifndef::tf-defaults[]
 +
-[IMPORTANT]
-====
-After you enter the name of the ROSA cluster and confirm destruction by entering `yes`, you cannot stop the `terraform destroy` process. Your account, Operator roles, and cluster are deleted.
-====
-
-. Enter the name of the cluster that you want to delete:
+The Terraform interface prompts you for four variables. These should match the answers you provided when creating a cluster:
 +
 [source,terminal]
 ----
-var.cluster_name
-  Provide the name of your ROSA cluster.
+var.admin_password
+  Set the password for the admin user
+
+  Enter a value:
+
+var.admin_username
+  Set the username for the admin user
 
-  Enter a value: <name_of_rosa_cluster> <1>
+  Enter a value:
+
+var.create_vpc
+  If you would like to create a new VPC, set this value to 'true.' If you do not want to create a new VPC, set this value to 'false.'
```

Comment:
Maybe slightly pedantic, but maybe but the '.' outside of the quotes? So it's "set this value to 'true'." instead of "set this value to 'true.'", as the full stop isn't valid syntax